### PR TITLE
[Pac] - Optimization + minor fixes.

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: libse/SubtitleFormats/Pac.cs [hunk 1]
```diff
@@ -955,12 +956,14 @@ private int AutoDetectEncoding()
                     if (count == 2)
                     {
                         _codePage = CodePageLatin;
-                        var sb = new StringBuilder("ABCDEFGHIJKLMNOPPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz(1234567890, .!?-\r\n'\"):;&");
+                        var sb = new StringBuilder("ABCDEFGHIJKLMNOPPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz¿¡" + ignoreChars);
                         foreach (var code in LatinCodes.Values)
                             sb.Append(code.Character);
                         var codePageLetters = sb.ToString();
+
+                        string textNoTags = HtmlUtil.RemoveHtmlTags(p.Text, true);
```
Comment (ivandrofly) [inline libse/SubtitleFormats/Pac.cs]: Remove html tags once

File: libse/SubtitleFormats/Pac.cs [hunk 2]
```diff
@@ -946,6 +946,7 @@ private int AutoDetectEncoding()
                 int index = 0;
                 int count = 0;
                 _codePage = CodePageLatin;
+                const string ignoreChars = "[](1234567890, .!?-\r\n'\"):;&";
```
Comment (ivandrofly) [inline libse/SubtitleFormats/Pac.cs]: cache this + add "[" and "]"

File: libse/SubtitleFormats/Pac.cs [hunk 3]
```diff
@@ -955,12 +956,14 @@ private int AutoDetectEncoding()
                     if (count == 2)
                     {
                         _codePage = CodePageLatin;
-                        var sb = new StringBuilder("ABCDEFGHIJKLMNOPPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz(1234567890, .!?-\r\n'\"):;&");
+                        var sb = new StringBuilder("ABCDEFGHIJKLMNOPPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz¿¡" + ignoreChars);
```
Comment (ivandrofly) [inline libse/SubtitleFormats/Pac.cs]: add spanish inverted question and exlcamation marks

File: libse/SubtitleFormats/Pac.cs (hunk 2 above)
Comment (niksedk) [inline libse/SubtitleFormats/Pac.cs]: :)

File: libse/SubtitleFormats/Pac.cs (hunk 1 above)
Comment (niksedk) [inline libse/SubtitleFormats/Pac.cs]: "p" is changed, so this will make AutoDetectEncoding useless.

**Pull request closed without merging**